docs/src/pages/components/alert/SimpleAlert.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Box } from '@material-ui/core';
+import { Alert as MuiAlert, AlertTitle, AlertContent } from '@material-ui/lab';
+import { withStyles } from '@material-ui/styles';
+
+/* eslint-disable react/jsx-filename-extension */
+
+const Wrapper = withStyles({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+})(Box);
+
+const Alert = withStyles({
```

Comment:
I think that we can merge these two withStyles into a single statement (to make it easier to copy and paste in a codebase). Something like this:
```jsx
import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';

const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    root: {
      '& > * + *': {
        marginTop: theme.spacing(2),
      },
    },
  }),
);
```
/docs/src/pages/components/progress/CircularDeterminate.tsx